src/App.tsx

```diff
@@ -1,19 +1,148 @@
 import React from 'react';
-import './App.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './App.scss';
 
+import { uuid } from 'uuidv4';
 import users from './api/users';
+import todos from './api/todos';
+import { TodoList } from './components/TodoList';
+import { SelectUser } from './components/SelectUser';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-};
+const todosWithUsers: Todo[] = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId) || null,
+}));
+
+interface State {
+  todos: Todo[];
+  users: User[];
+  todoTitle: string;
+  selected: string;
+  isFilled: boolean;
+  isSelected: boolean;
+}
+
+class App extends React.Component<{}, State> {
+  state = {
+    todos: todosWithUsers,
+    users,
+    todoTitle: '',
```

Comment:
I am not sure if you need to keep users in the state. They have storage on API, so you can use them without keeping them in state